lib/features/search/search.js

```diff
@@ -27,218 +27,242 @@
  * @returns {SearchResults}
  */
 export default function search(items, pattern, options) {
-  return items.reduce((results, item) => {
-    const tokens = getTokens(item, pattern, options.keys);
 
-    if (Object.keys(tokens).length) {
-      const result = {
-        item,
-        tokens
-      };
+  const {
+    keys
+  } = options;
 
-      const index = getIndex(result, results, options.keys);
+  const words = pattern.trim().toLowerCase().split(/\s+/);
 
-      results.splice(index, 0, result);
+  return items.flatMap((item, idx) => {
```

Comment:
`idx` seems not to be used anywhere

Reply:
Fixed via https://github.com/bpmn-io/diagram-js/pull/932/commits/5d1d75690023f794c9e617cc8921535c3e91b196.